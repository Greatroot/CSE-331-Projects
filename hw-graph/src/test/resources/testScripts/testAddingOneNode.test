#Test adding one node | should appear within the graph's nodes.

CreateGraph graph
AddNode graph node