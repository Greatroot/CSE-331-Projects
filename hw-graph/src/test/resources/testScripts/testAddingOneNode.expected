#Test adding one node | should appear within the graph's nodes.

created graph graph
added node n1 to graph